tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/Request.cs

```diff
@@ -29,9 +29,16 @@ public async Task PutJsonAsync(string url, object payload, params object[] pathS
 
         private IFlurlRequest CreateCommonRequest(string url, params object[] pathSegments)
         {
+            string accessToken = null;
```

Comment:
This used to be string.Empty but as long as the tests pass & the method `WithOAuthBearerToken` can accept a `null` looks good 👍